apps/laboratory/src/utils/SiweUtils.ts

```diff
@@ -4,17 +4,17 @@ import type { SIWEVerifyMessageArgs, SIWECreateMessageArgs, SIWESession } from '
 import { createSIWEConfig } from '@web3modal/siwe'
 
 export const siweConfig = createSIWEConfig({
-  createMessage: ({ nonce, address, chainId }: SIWECreateMessageArgs) =>
-    new SiweMessage({
-      version: '1',
-      domain: window.location.host,
-      uri: window.location.origin,
-      address,
-      chainId,
-      nonce,
-      // Human-readable ASCII assertion that the user will sign, and it must not contain `\n`.
-      statement: 'Sign in With Ethereum.'
-    }).prepareMessage(),
+  messageParams: {
+    domain: 'lab.web3modal.com',
+    uri: 'https://lab.web3modal.com',
+    chains: [1, 56],
+    resources: [
+      'https://siwe.dev',
+      'urn:recap:eyJhdHQiOnsiaHR0cHM6Ly93ZWIzaW5ib3guY29tIjp7InB1c2gvYWxlcnRzIjpbe31dLCJwdXNoL25vdGlmaWNhdGlvbnMiOlt7fV19fX0='
+    ],
+    statement: 'Test statement'
```

Comment:
Could you add context around resources and the `urn` ? Perhaps by using a variable with an explicit name ?